### PR TITLE
update node cli commands

### PR DESCRIPTION
**Pull request by jcstein**
## Overview

This PR adds:
- [x] transfer endpoint commands
- [x] `celestia-node` CLI commands
- [x] `./cel-key` CLI commands
- [x] import wallet from mnemonic command.

## Checklist

- [x] New and updated code has appropriate documentation
- [x] New and updated code has new and/or updated testing
- [x] Required CI checks are passing
- [x] Visual proof for any user facing features like CLI or documentation updates
- [x] Linked issues closed with keywords

### CODE REVIEW

**Comment by Bidon15** [comment]
Considering that we have new arabica versions, now the default rest api port is `26659` 🙏 
Thanks for the PR @jcstein 💪

**Comment by jcstein** [comment]
thank you @Bidon15 - added 26659 across that doc

File: docs/developers/node-api.md [hunk 1]
```diff
@@ -679,3 +679,324 @@ Response
...
+          "value": "YmFuaw==",
+          "index": true
+        }
+      ]
+    }
+  ]
+}
+```
+
+## Helpful CLI Commands
```
Comment (YazzyYaz) [inline docs/developers/node-api.md:900]: I'd probably move this section to a separate doc.

We are probably going to eventually phase out this doc for Node API reference when OpenRPC implementation is ready.

Comment (jcstein) [reply docs/developers/node-api.md:900]: sounds good, will add a page for "helpful CLI commands" for node, similar to app

**Comment by jcstein** [comment]
i found that there is a cel-key utility guide in the `node` section of docs and just made changes to reflect this. this should simplify the PR significantly

File: docs/nodes/keys.mdx [hunk 2]
```diff
@@ -0,0 +1,243 @@
+---
+sidebar_label: Keys
```
Comment (YazzyYaz) [inline docs/nodes/keys.mdx:2]: Check the latest main, we have renamed this file to `[celestia-node-key](https://docs.celestia.org/developers/celestia-node-key)`

I'd update this PR to be based on the updates inside the new file name

Comment (jcstein) [reply docs/nodes/keys.mdx:2]: should be good in my last push, updated with the tabs i made on the new page

**Review by YazzyYaz — APPROVED** [review]
LGTM

**Pull request merged**